--- download/vocab/https:/raw.githubusercontent.com/w3c/vc-data-model/main/vocab/credentials/credentials.ttl	2022-12-24 23:23:09.773127287 +0000
+++ credentials.ttl	2022-12-24 23:23:04.189116239 +0000
@@ -11,8 +11,8 @@
   dc:title "Verifiable Credentials Vocabulary"@en;
   dc:description """This document describes the RDFS vocabulary description used for Verifiable Credentials [[VERIFIABLE-CREDENTIALS]] along with the default JSON-LD Context."""@en;
   dc:date "2019-05-26"^^xsd:date;
-  dc:imports ;
   rdfs:seeAlso <https://www.w3.org/TR/vc-data-model/>;
+.
 
 # Class definitions
 cred:JsonSchemaValidator2018 a rdfs:Class;
